### ISSUE TITLE
Model saving issue

### ISSUE THREAD

**Issue by lindadamama** [dotnet/TorchSharp#1463]
I'm currently developing a semantic segmentation model and have encountered an issue where the torchsharp model  to save. When I use the model that is in the training process for inference, it operates normally. However, when I save the model to the hard drive and reload it, the returned result is all zeros.
Storage method
 module.eval();
 //torch.save(module.state_dict(), "model_weights.pt");
 module.save(saveModelName);

Load method

 module = new SeaFormerSegmentation(device, numClass, modelType.base1);
 try 
 {
     if (File.Exists(modelPtah))
     {
         module.load(modelPtah);
     }

 } catch { }

**Comment by lindadamama** [comment]
Write it like this, it returns a result, I don't know why, it can't be replaced automatically?              
 module.eval();
                if (File.Exists(modelPtah)) 
                {
                    File.Delete(modelPtah);
                }
                //torch.save(module.state_dict(), "model_weights.pt");
                module.save(modelPtah);

**Comment by lindadamama** [comment]
Convergence effect

<img width="851" alt="Image" src="https://github.com/user-attachments/assets/43ced5bf-ffcd-4986-ae77-e0606691a07a" />

Model saving method：
``
  module.save(saveModelName);

Encountered two problems
1. The Model.eval() method, which cannot mask all Dropout methods, still works after being called.

2, is the problem of model saving, as mentioned above, directly pointing to the model being trained, the result is roughly correct, but save to the hard disk and reload, the result is completely incorrect

The source code is as follows

[conde.zip](https://github.com/user-attachments/files/19402803/conde.zip)

**Comment by alinpahontu2912** [comment]
Hey @lindadamama, I think it might be related to this: https://github.com/dotnet/TorchSharp/issues/1426. There is a fix for it, but the release containing it will take a bit more time.
